ibind/client/ibkr_client.py

```diff
@@ -63,10 +87,392 @@ def __init__(
             url = f'https://{host}:{port}{base_route}'
 
         self.account_id = account_id
+        self._use_oauth=use_oauth
+        # if self._use_oauth:
+        #     self.live_session_token,self.access_token=self.req_live_session_token()
+        # else:
+        #     self.live_session_token,self.access_token=None,None
         super().__init__(url=url, cacert=cacert, timeout=timeout, max_retries=max_retries)
 
         self.logger.info('#################')
         self.logger.info(f'New IbkrClient(base_url={self.base_url!r}, account_id={self.account_id!r}, ssl={self.cacert!r}, timeout={self._timeout}, max_retries={self._max_retries})')
 
+
+
     def make_logger(self):
         self._logger = new_daily_rotating_file_handler('IbkrClient', os.path.join(var.LOGS_DIR, f'ibkr_client_{self.account_id}'))
+        
+    def req_live_session_token(self) -> tuple[str, int]:
+        """Get live session token and access token from IBKR Web API used to make API endpoint calls
+
+        Raises:
+            Exception: _description_
+            Exception: _description_
+
+        Returns:
+            tuple[token,token]: live access token, access token
+        """
+
+        REQUEST_URL = "https://api.ibkr.com/v1/api/oauth/live_session_token"
+        REQUEST_METHOD = "POST"
+        ENCRYPTION_METHOD = "RSA-SHA256"
+
+        
+        load_dotenv()
+        config = configparser.ConfigParser()
+        config.read('D:\\git_repos\\oauth_env\\oauth_test.env')
+
+        consumer_key = config['consumer_key']['CONSUMER_KEY']
+        access_token_secret=config['access_token_secret']['ACCESS_TOKEN_SECRET']
+        encription_key_fp=config['keys']['ENCRYPTION_KEY_FP']
+        dh_prime_fp=config['Diffie_Hellman']['DH_PRIME_FP']
+        dh_generator=config['Diffie_Hellman']['DH_GENERATOR']
+        realm=config['realm']['REALM']
```

Comment:
It's very cool to see you show initiative and come up with a way to do this thought ConfigParser 👍 However, there already is a standard way of reading configuration, and these are environment variables. If you'd feel there are any reasons why using ConfigParser would be superior let me know, otherwise see `var.py` and let's adapt these.

Reply:
I agree, I use ConfigParser in my other code and used it here to save time.
I usually get the code to work, and then tidy up.

Reply:
And that's perfectly reasonable 👍 like I say, I appreciate the initiative here, and yes, tidying up at the end makes sense. All good 👏